types/vscode/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Visual Studio Code 1.34
+// Type definitions for Visual Studio Code 1.35
 // Project: https://github.com/microsoft/vscode
 // Definitions by: Visual Studio Code Team, Microsoft <https://github.com/Microsoft>
```

Comment:
@kieferrm, if you list individual owners (e.g. yourself) in Definitions By, the PR will get fast-tracked as owner-approved.  Of course, that will be moot once @sandersn confirms you have merge permission. :smile: